cmd/helpers.go

```diff
@@ -46,3 +49,34 @@ func UserConfirmYes(ctx *cmd.Context) error {
 	}
 	return nil
 }
+
+// UserConfirmName returns an error if we do not read a "name" of the model/controller/etc from user
+// input.
+func UserConfirmName(verificationName string, objectType string, ctx *cmd.Context) error {
+	fmt.Fprintf(ctx.Stdout, "\nTo continue, enter the name of the %s to be destroyed:", objectType)
+	scanner := bufio.NewScanner(ctx.Stdin)
+	scanner.Scan()
+	err := scanner.Err()
+	if err != nil && err != io.EOF {
+		return errors.Trace(err)
+	}
+	answer := strings.ToLower(scanner.Text())
+	if answer != verificationName {
+		return errors.Trace(userAbortedError("aborted"))
+	}
+	return nil
+}
+
+// CheckSkipConfirmEnvVar check if JujuSkipConfirmationEnvKey environment variable is defined, and if yes -
+// return it's boolean value
+func CheckSkipConfirmEnvVar() (bool, error) {
+	if envSkipConfirmValueStr := os.Getenv(osenv.JujuSkipConfirmationEnvKey); envSkipConfirmValueStr != "" {
```

Comment:
Another knitpick, might be worth using LookupEnv here. this will return an err if the var is unset, instead of just an empty string. This will resolve the case where a set but empty JUJU_SKIP_CONFIRMATION var fails silently

Reply:
Another knit-pick, but this function is slightly misleading. If the env var is unset, it assumes it's value should be false. 

Imo this is outside the responsibility of this function. To know what the default value should be, that is. We are just checking it afterall. I think, in the case of an unset env var, an error should be returned. The caller can then decide what to do (i.e. don't overwrite c.assumeNoPrompt)

Reply:
Agreed, fixed